nixos/modules/services/cluster/kubernetes/default.nix

```diff
@@ -279,6 +296,8 @@ in {
       # dns addon is enabled by default
       services.kubernetes.addons.dns.enable = mkDefault true;
 
+      environment.systemPackages = [pkgs.containerd];
```

Comment:
This probably need to be moved to the `containerd` module.

Reply:
Done (ff10d964fef1a459c2bdc3493c4dbc84151adfea)